pom.xml

```diff
@@ -1300,7 +1300,12 @@
             <dependency>
                 <groupId>commons-pool</groupId>
                 <artifactId>commons-pool</artifactId>
-                <version>1.6</version>
```

Comment:
This change isn't technically necessary as dependencyManagement in gs-plugin and web modules takes care of making sure we use the provided one (1.5.3) in geoserver WEB-INF/lib.  If issue for cq or preferred, can set back to 1.6.